### ISSUE TITLE
Is there support for TS.GET?

### ISSUE THREAD

**Issue by ukd1** [jgaskins/redis#11]
Is there support for [TS.GET](https://redis.io/commands/ts.get/) - I can't find it, but might be missing something obvious. Thanks for building this @jgaskins!

**Comment by jgaskins** [comment]
Unfortunately, not yet. Looks like it's still missing support for [several `TS.*` commands](https://jgaskins.dev/redis/Redis/TimeSeries.html). I thought I had built more of it out, tbh, but maybe that was just a fever dream. 😄

On the bright side, since everything is implemented in terms of `run` (see the note at the top of the [`Redis::Commands` docs](https://jgaskins.dev/redis/Redis/Commands.html)), you can indeed get that using `Redis::Client#run`:

```crystal
result = redis.run({"ts.get", key})
# => [1656298858190, "397.0992473379659"]
```

---

### Related context not specifically asked for

I'd love to expand support for `TimeSeries` and make it easier to use in general. Casting the array results for some of these Redis modules is not very expressive, especially when they're nested. It's similar for streams, so I added a few types in [`Redis::Streaming`](https://github.com/jgaskins/redis/blob/master/src/streaming.cr) that can deconstruct stream responses from Redis. We could probably use the same idea here:

```crystal
struct Redis::TimeSeries::GetResponse
  getter timestamp : Time
  getter value : Float64

  def initialize(result)
    result = result.as(Array)
    @timestamp = Time.unix_ms(result[0].as(Int))
    @value = result[1].as(String).to_f64
  end
end

pp Redis::TimeSeries::GetResponse.new(redis.ts.get(key))
# Redis::TimeSeries::GetResponse(
#  @timestamp=2022-06-27 03:06:50.359000000 UTC,
#  @value=155.89755024739296)
```

I don't think I've documented this, but the client deals mostly with raw Redis responses so that commands can be used on [all types that include the `Redis::Commands` mixin](https://jgaskins.dev/redis/Redis/Commands.html): `Connection`, `Cluster`, `Pipeline`, and `Transaction`. Time-series commands are an excellent use case for both pipelined commands so you only pay the latency cost once and on Redis clusters to spread out storage of related metrics:

```crystal
redis.pipeline do |pipe|
  pipe.ts.add "latency:#{labels}", value: latency, on_duplicate: :last, labels: labels
  pipe.ts.add "throughput:#{labels}", value: 1, on_duplicate: :sum, labels: labels
end

# Using "{latency}:..." to store all latency metrics on the same server in the cluster
cluster.ts.add "{latency}:#{labels}", value: latency
```

This may not work as written right now, but that's definitely the goal.

**Comment by ukd1** [comment]
Thanks @jgaskins - I ended up reading the code and coming up with the way you suggested, along with some checking, etc. I'll checkout the further details tomorrow! Thanks for quick reply 👍🏻

**Comment by jgaskins** [comment]
Fixed in 78b7a22
